src/components/landing-page/LandingComponent.tsx

```diff
@@ -1,9 +1,55 @@
 import React from 'react';
+import StarShopLanding from "../../../public/starshop-logos/StarShop-Logo-Landing.svg"
+import Link from 'next/link'
+import Image from "next/image"
 
 const LandingPageComponent: React.FC = () => {
   return (
     <>
-      <h1 className='text-white text-2xl font-bold mt-8'>Landing page</h1>
+      <div className='text-white h-[120px]'>
+        Add header here
+      </div>
```

Comment:
```suggestion
      <div className='text-white h-[120px]'>
        Add header here
      </div>
```
Delete this part of the code please

Reply:
Fixed!